internal/convert_test.go

```diff
@@ -50,28 +50,47 @@ func TestGetDDL(t *testing.T) {
 		ColNames: []string{"a", "b"},
 		ColDefs: map[string]ddl.ColumnDef{
 			"a": ddl.ColumnDef{Name: "a", T: ddl.Type{Name: ddl.Int64}},
-			"b": ddl.ColumnDef{Name: "b", T: ddl.Type{Name: ddl.Float64}},
+			"b": ddl.ColumnDef{Name: "b", T: ddl.Type{Name: ddl.Int64}},
 		},
-		Pks: []ddl.IndexKey{ddl.IndexKey{Col: "a"}}}
+		Pks: []ddl.IndexKey{ddl.IndexKey{Col: "a"}},
+		Fks: []ddl.Foreignkey{ddl.Foreignkey{Name: "fk1", Columns: []string{"b"}, ReferTable: "ref_table1", ReferColumns: []string{"ref_c"}}},
```

Comment:
just a very minor nit, why do we refer to ref_c and not ref_b here for the sake of consistency?

Reply:
renamed to ref_b